### PR TITLE
add HTML iframe support

### PR DESCRIPTION
**Pull request by KolombetM**
Added extra view class with css to remove margins

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/u2/navigation/IFrameTopNavigation.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
```
Comment (jlhughes) [inline src/foam/nanos/u2/navigation/IFrameTopNavigation.js]:
```suggestion
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
```

Comment (kgrgreer) [inline src/foam/nanos/u2/navigation/IFrameTopNavigation.js]: 2019

File: src/foam/comics/DAOControllerView.js [hunk 2]
```diff
@@ -212,6 +213,15 @@ foam.CLASS({
           .end());
 
       this.add(this.data.border);
+      if (this.isIframe()) this.tag(this.IFrameTopNavigation);
+    },
+
+    function isIframe () {
+      try {
+        return window.self !== window.top;
```
Comment (kgrgreer) [inline src/foam/comics/DAOControllerView.js:221]: Better to import 'window' then do this.window. Makes sure that components can work with multiple windows.

File: src/foam/comics/DAOControllerView.js [hunk 3]
```diff
@@ -212,6 +213,15 @@ foam.CLASS({
           .end());
 
       this.add(this.data.border);
+      if (this.isIframe()) this.tag(this.IFrameTopNavigation);
```
Comment (kgrgreer) [inline src/foam/comics/DAOControllerView.js]: spacing:
if ( this.isIframe() ) this.tag(this.IFrameTopNavigation);
Please read the FOAM style guide: https://github.com/foam-framework/foam/wiki/CodingStyle

**Comment by kgrgreer** [comment]
Good job, looks good. Just fix year and spacing.

**Pull request merged**